src/meson.build

```diff
@@ -42,6 +42,7 @@ src_files += files(
     'input_event.hpp',
     'utils.hpp',
     'utils.cpp',
+    'src/command_line_arguments.hpp',
```

Comment:
remove src/

Reply:
execute this command in the root `meson wrap install argparse`

also in the root meson file add this:

`
deps += dependency('argparse', required: true, fallback: ['argparse'])
`
(after `deps += dependency('magic_enum', required: true, fallback: ['magic_enum'])`)

Reply:
> remove src/

Everytime 🙈